packages/qwik/src/qwikloader.ts

```diff
@@ -102,10 +102,12 @@ export const qwikLoader = (
       for (const qrl of attrValue.split('\n')) {
         const url = new URL(qrl, base);
         const href = url.href;
-        const symbol = url.hash[replace](/^#?([^?[|]*).*$/, '$1') || 'default';
+        const match = /^#?(([^#]+)#)?([^?[|]*).*$/.exec(url.hash);
```

Comment:
can we add regexp online editor link. for example https://regexr.com/ lets you also add regexp tests for example https://regexr.com/816es (don't use mine we should make a qwik account with all them saved